shared/js/background/chrome-events.es6.js

```diff
@@ -132,6 +136,9 @@ chrome.runtime.onMessage.addListener((req, sender, res) => {
         let value = req.updateSetting['value']
         settings.ready().then(() => {
             settings.updateSetting(name, value)
+            if (name === 'trackerBlockingEnabled') {
+                tabManager.handleTrackerBlockingToggle(value)
```

Comment:
All tracker blocking enable disabling goes through here (on post install page, in extension popup, on settings page). When this setting is changed, the state of all popups needs to be updated.